src/__mocks__/datasource.ts

```diff
@@ -8,6 +8,7 @@ export const mockDatasource = new DataSource({
   type: 'redshift-datasource',
   name: 'Redshift Data Source',
   jsonData: {},
+  access: 'direct',
```

Comment:
Not relevant to long running queries, but after updating the dev deps, this was added to fix an error.